index.html

```diff
@@ -550,16 +553,19 @@ <h2>Group 1</h2>
             <h2>Group 2</h2>
             <div class="sub-footer-link" role="link">
               <a href="./privacy-policy.html">Privacy Policy</a>
+              <br>
               <a href="terms">Terms</a>
+              <br>
               <a href="./faq.html">FAQ</a>
+              <br>
               <a href="./support.html">Support</a>
             </div>
           </section>
 
           <section class="footer-column">
             <h2>About Us</h2>
             <div>
-              Keiko Corp. the greatest start up of all time Bruno the CEO of
+              Keiko Corp. The greatest start up of all time. Bruno,The CEO of
```

Comment:
👏 Thanks for looking this over, here is some feedback!

"Keiko Corp. **the** greatest start up of all time. **Bruno, the** CEO of Keiko **Corp.,** has built the fastest growing tech start up in Silicon Valley."

Corp. is an abbreviation for Corporation, as such it's not a '.' as used in sentence punctuation.
"Bruno, the" not "Bruno, The" we would only capitalize the T at the start of a new clause, which in this case it is not!

The period after time, should be there: good eye!

Reply:
Thank you for the feedback .This is my first open source contribution. I will change it according to your feedback.

The comma after Bruno, Is this correct?

Reply:
The socials in this footer block are also not visibl
<img width="960" alt="keiko-corp" src="https://user-images.githubusercontent.com/84712380/136126088-7e23af93-92c9-4726-9836-69255fbb1b9c.png">
e

Reply:
Can I make another pull request resolving this issues and the feedback you have given for me?
<img width="960" alt="keiko-corp-updated" src="https://user-images.githubusercontent.com/84712380/136126528-671412b1-823b-432a-90ec-a1f02f5a1191.png">